packages/flutter_tools/test/src/android_common.dart

```diff
@@ -156,33 +161,35 @@ Future<ProcessResult> buildFlutterApkWithSpecifiedDependencyVersions({
   ], workingDirectory: tempDir.path);
   expect(result, const ProcessResultMatcher());
 
-  final Directory app = Directory(fileSystem.path.join(tempDir.path, 'dependency_checker_app'));
+  final Directory app =
+      Directory(fileSystem.path.join(tempDir.path, 'dependency_checker_app'));
 
   if (versions.compileSdkVersion != null) {
-    final File appGradleBuild = File(fileSystem.path.join(
-        app.path, 'android', 'app', 'build.gradle'));
-    final String appBuildContent = appGradleBuild.readAsStringSync()
+    final File appGradleBuild = File(
+        fileSystem.path.join(app.path, 'android', 'app', 'build.gradle.kts'));
+    final String appBuildContent = appGradleBuild
+        .readAsStringSync()
         .replaceFirst(flutterCompileSdkString, versions.compileSdkVersion!);
```

Comment:
I think `appBuildContent` looks better now than it did before 👍

`appGradelBuild` should probably be changed to:

```dart
    final File appGradleBuild = File(
      fileSystem.path.join(app.path, 'android', 'app', 'build.gradle.kts'),
    );
```